canonicalwebteam/discourse_docs/app.py

```diff
@@ -70,7 +70,8 @@ def document_view(path=""):
                     return flask.abort(http_error.response.status_code)
 
                 if category_id and topic["category_id"] != category_id:
-                    return flask.abort(404)
+                    forum_topic_url = api.base_url + "/t/" + path
```

Comment:
This is a bit hacky. It won't work well if you `path` is `/t/topic/1234`. I think a better solution is to move line 76 (`document = parse_topic(topic)`) above this conditional, and then use `f"{api.base_url}{document['topic_path']}"`, like we do [in the templates](https://github.com/canonical-web-and-design/docs.jujucharms.com/blob/master/templates/document.html).